crowbar_framework/app/models/service_object.rb

```diff
@@ -1279,6 +1395,8 @@ def apply_role(role, inst, in_queue)
       end
 
       # Make sure the config role is on the nodes in this barclamp, otherwise remove it
+      # FIXME: All nodes is basically all new nodes, which should have an add/remove list,
+      # why do we need to add specifically the passed role?
```

Comment:
It is not clear to me, why do we need to explicitly add the role passed to the `apply_role` - wasn't it already added via `alist` a few lines above?